### ISSUE TITLE
/lib/mjs/plugin module not found

### ISSUE THREAD

**Issue by lipkiy** [climba03003/fastify-bree#87]
Hi. I'm catching next error when starting app:

```
node:internal/errors:465
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/usr/app/node_modules/fastify-bree/lib/mjs/plugin' imported from /usr/app/node_modules/fastify-bree/lib/mjs/index.js
    at new NodeError (node:internal/errors:372:5)
    at finalizeResolution (node:internal/modules/esm/resolve:437:11)
    at moduleResolve (node:internal/modules/esm/resolve:1009:10)
    at defaultResolve (node:internal/modules/esm/resolve:1218:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

Fastify-bree v3.0.3
Node v16.18 (alpine docker image)
Fastify v4.9.2

Fixed that by changing line 4 in "lib/mjs/index.js" from:
`import { BreeTS } from './plugin';`

To:
`import { BreeTS } from './plugin.js';`

**Comment by climba03003** [comment]
Strange, shouldn't the extension be auto fill by node itself

**Comment by lipkiy** [comment]
> Strange, shouldn't the extension be auto fill by node itself

Yep, but it doesn't do it. IDK why.

**Comment by climba03003** [comment]
Fixed in `3.0.4`
